model/news.model.js

```diff
@@ -43,3 +44,19 @@ exports.fetchArticleById = (article_id) => {
     msg: "invalid sort query!",
   });
 };
+
+exports.fetchCommentsById = (article_id) => {
+  if (!isNaN(article_id)) {
+    return checkIfArticleExists(article_id)
+      .then(() => {
+        return db.query(
+          `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;`,
+          [article_id]
+        );
+      })
+      .then((result) => {
+        return result.rows;
+      });
+  }
+  return Promise.reject({ status: 400, msg: "invalid sort query!" });
+};
```

Comment:
It might be a bit clearer if you move this Promise.reject up to directly under line 49.  You can phrase it like:
'if article_id is not a number, reject the promise'